website/docs/foundations/focus-ring/index.md

```diff
@@ -4,8 +4,94 @@ caption: Focus rings are a vital tool in communicating interactivity.
 description: Focus rings display when interactive components including links and buttons are focused by the user. They are a vital tool in communicating interactivity.
 ---
 
-<section data-tab="Code">
-  @include "partials/code/how-to-use.md"
-  @include "partials/code/showcase.md"
-</section>
+## How to use this style
 
+The suggested way to apply a "focus-ring" style to an UI element is using the specific **design token** provided as CSS custom property.
+
+### Design tokens
+
+You can use the `--token-focus-ring-action-box-shadow` [design token](./tokens) directly in your CSS definitions:
+
+```css
+.your-selector {
+  [...your CSS declarations]
+  &:focus,
+  &:focus-visible {
+    box-shadow: var(--token-focus-ring-action-box-shadow);
+  }
+}
+```
+
+!!! Critical
+
+*   The design token as CSS variable can be used **only** with a `box-shadow` property
+*   The border radius depends on the UI element to which is applied to, so it's up to you to apply the right `border-radius` (tip: consider to use the `inherit` value).
+!!!
+
+### CSS helper classes
+
+We provide also a **CSS helper class** `.hds-focus-ring-box-shadow`, that is a wrapper of the design token. It's unlikely you can use it directly in a template because this style is connected to the "focused" pseudo-state of an element (more likely it would be used in composition with other classes).
```

Comment:
```suggestion
We provide a **CSS helper class** `.hds-focus-ring-box-shadow`, that is a wrapper of the design token. It's unlikely you can use it directly in a template because this style is connected to the "focused" pseudo-state of an element (more likely it would be used in composition with other classes).
```